### PR TITLE
One more developers page: how to setup personal devnet in local

### PR DESCRIPTION
**Pull request by ws4charlie**
1. Added a page to help developers to setup their own devnet in local.

### CODE REVIEW

File: docs/developers/local-devnet.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 1
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Personal Devnet
```
Comment (kingpinXD) [inline docs/developers/local-devnet.md:7]:
```suggestion
# How to start a local network
```

**Comment by dlin128** [comment]
I get a failure on make build_release:
```
XXX platform % make build_release
====== Build root is "target" ======
rm 
usage: rm [-f | -i] [-dIPRrvWx] file ...
       unlink [--] file
make: [tendermint_cleveldb] Error 64 (ignored)
bash tools/download_tendermint.sh 'tools/tendermint'
mkdir -p /Users/xxx/go/bin
cd tools/tendermint \
		&& /Applications/Xcode.app/Contents/Developer/usr/bin/make build TENDERMINT_BUILD_OPTIONS=cleveldb \
		&& cp build/tendermint /Users/xxx/go/bin/
CGO_ENABLED=1 go build -mod=readonly -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD` -s -w " -trimpath -tags 'tendermint cleveldb' -o build/tendermint ./cmd/tendermint/
build flag -mod=readonly only valid when using modules
make[1]: *** [build] Error 1
make: *** [tendermint_cleveldb] Error 2
```

**Comment by dlin128** [comment]
I've made a bunch of updates to the instructions for linux, but still cannot build on Mac OS X 12.4.

**Comment by dlin128** [comment]
Fixed by updating golang.org/x/sys

**Review by dlin128 — APPROVED** [review]

File: docs/developers/local-devnet.md (hunk 1 above)
Comment (ws4charlie) [reply docs/developers/local-devnet.md:9]: Is `Personal Findora Blockchain` good?

**Pull request merged**